docs/docs/concepts/recipes.md

```diff
@@ -3,16 +3,165 @@ title: Recipes
 sidebar_position: 2
 ---
 
-To express how a model is stored and presented, we use a concept of recipes. Every blueprint can have two kind of recipes in the Data Modelling Tool.
+To express how a model is _stored_ and how it is _presented_, we use a concept of StorageRecipes and UiRecipes.
 
-A recipe is just a specific instance of a recipe blueprint, and Libraries, plugins and services can use these recipes to make decisions and therefore become easy to replace.
+A recipe is an instance of the core Storage-/UiRecipe blueprints.
 
-## Storage Recipe
+## StorageRecipe
 
-A recipe for how to store the data for a blueprint.
+A recipe for how to store the data for an entity based on the given blueprint type.
 
-Contained means the value is stored in-place, and all primitives are stored with default contained property. However, if a blueprint is referring to another blueprint, then that blueprint can be stored as own independent entity.
+StorageRecipes are mainly to be consumed by the Data Modelling Storage Service.  
 
-## UI Recipe
+For now, StorageRecipes are mainly used to split large models into smaller parts. Either because it makes sense logically, or for performance reasons.
 
-A recipe for how to present data.
\ No newline at end of file
+A StorageRecipes may look something like this;
+
+```json
+{
+  "type": "dmss://system/SIMOS/StorageRecipe",
+  "name": "DEFAULT",
+  "description": "",
+  "attributes": [
+    {
+      "name": "content",
+      "type": "object",
+      "contained": false
+    }
+  ]
+}
+```
+
+This StorageRecipes is used for the core `dmss://system/SIMOS/Package`-type. Here we can see that any objects that are a 
+part of the "content" attribute for a package, are not contained in that package entity, but is only referred to. Which makes sense logically.
```

Comment:
```suggestion
part of the "content" attribute for a package, is not contained in that package entity but is only referred to. Which makes sense logically.  Instead of the actual object, a reference object is added, and it includes the name, type, and _id of the object referenced. 
```